kie-wb-smoke-tests/pom.xml

```diff
@@ -385,6 +391,27 @@
                 <groups>org.kie.smoke.wb.category.KieWbSeleniumSmoke</groups>
               </configuration>
             </plugin>
+            <plugin> <!-- Download & unzip IEDriverServer.exe - a binary required to run InternetExplorerDriver -->
```

Comment:
To make the `<execution>` work it needs to be moved directly to `<plugins>` section of the POM (instead of `<pluginManagement>`)